plugins/rest-plugin/src/main/java/ru/sbtqa/tag/api/EndpointEntryReflection.java

```diff
@@ -0,0 +1,161 @@
+import ru.sbtqa.tag.api.annotation.Body;
+import ru.sbtqa.tag.api.annotation.FromResponse;
+import ru.sbtqa.tag.api.annotation.Header;
+import ru.sbtqa.tag.api.annotation.ParameterType;
+import ru.sbtqa.tag.api.annotation.Query;
+import ru.sbtqa.tag.api.annotation.Stashed;
+import ru.sbtqa.tag.api.annotation.Validation;
+import ru.sbtqa.tag.api.annotation.applicators.Applicator;
+import ru.sbtqa.tag.api.annotation.applicators.ApplicatorHandler;
+import ru.sbtqa.tag.api.annotation.applicators.FromResponseApplicator;
+import ru.sbtqa.tag.api.annotation.applicators.StashedApplicator;
+import ru.sbtqa.tag.api.exception.RestPluginException;
+import ru.sbtqa.tag.api.properties.ApiConfiguration;
+import static ru.sbtqa.tag.api.utils.ReflectionUtils.get;
+import static ru.sbtqa.tag.api.utils.ReflectionUtils.set;
+import ru.sbtqa.tag.qautils.reflect.FieldUtilsExt;
+
+public class EndpointEntryReflection {
+
+    private static final Logger LOG = LoggerFactory.getLogger(EndpointEntryReflection.class);
+    private static final ApiConfiguration PROPERTIES = ConfigFactory.create(ApiConfiguration.class);
+    private static final String BOM = "\uFEFF";
+
+    private EndpointEntry endpoint;
+    private List<Field> fields;
+
+    public EndpointEntryReflection(EndpointEntry endpoint) {
+        this.endpoint = endpoint;
+        this.fields = FieldUtilsExt.getDeclaredFieldsWithInheritance(endpoint.getClass());
+    }
+
+    public void applyAnnotations() {
+        for (Field field : fields) {
+            field.setAccessible(true);
+
+            ApplicatorHandler<Applicator> applicators = new ApplicatorHandler<>();
+            for (Annotation annotation : field.getAnnotations()) {
+                if (annotation instanceof FromResponse) {
+                    applicators.add(new FromResponseApplicator(endpoint, field));
+                } else if (annotation instanceof Stashed) {
+                    applicators.add(new StashedApplicator(endpoint, field));
+                } else {
+                    continue;
+                }
+            }
+
+            applicators.apply();
+        }
+    }
+
+    public void setParameterValueByTitle(String name, String value) {
+        for (Field field : fields) {
+            for (Annotation annotation : field.getAnnotations()) {
+                if (annotation instanceof Body && ((Body) annotation).name().equals(name)
+                        || annotation instanceof Query && ((Query) annotation).name().equals(name)
+                        || annotation instanceof Header && ((Header) annotation).name().equals(name)
+                        && value != null && !value.isEmpty()) {
+                    set(endpoint, field, value);
+                    return;
+                }
+            }
+        }
+
+        throw new RestPluginException("There is no '" + name + "' parameter in '" + endpoint.getClass().getAnnotation(ru.sbtqa.tag.api.annotation.Endpoint.class).title() + "' endpoint.");
```

Comment:
1. Exteract variable of endpoint.getClass().getAnnotation(ru.sbtqa.tag.api.annotation.Endpoint.class).title()
2. use String.format